docs/docs/guides/FRAME_PROCESSOR_CREATE_PLUGIN_ANDROID.mdx

```diff
@@ -54,14 +55,17 @@ For reference see the [CLI's docs](https://github.com/mateusz1913/vision-camera-
 2. Create a Java source file, for the Face Detector Plugin this will be called `FaceDetectorFrameProcessorPlugin.java`.
 3. Add the following code:
 
-```java {8}
+```java
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
 import com.mrousavy.camera.frameprocessor.Frame;
 import com.mrousavy.camera.frameprocessor.FrameProcessorPlugin;
 
 public class FaceDetectorFrameProcessorPlugin extends FrameProcessorPlugin {
-
+  @Nullable
   @Override
-  public Object callback(Frame frame, Map<String, Object> arguments) {
+  public Object callback(@NonNull Frame frame, @Nullable Map<String, Object> arguments) {
```

Comment:
I think to keep this simple we can skip the `@NonNull` and `@Nullable` annotations here and in the return decl and in the imports - just less overhead for someone not as affine with native development.